### PR TITLE
meeting at half point

### PR DESCRIPTION
**Pull request by Galithil**
allows to add extra args to the recurse_for_every_seqrun_in_sample call, which can be used to reset coverage or amount of reads when relaunching a sample

### CODE REVIEW

File: ngi_pipeline/engines/piper_ngi/local_process_tracking.py [hunk 1]
```diff
@@ -176,7 +176,7 @@ def update_charon_with_local_jobs_status(config=None, config_file_path=None):
 
 @with_ngi_config
 def recurse_status_for_sample(project_obj, set_status, update_done=False,
-                              config=None, config_file_path=None):
+                              extra_args={},config=None, config_file_path=None):
```
Comment (mariogiov) [inline ngi_pipeline/engines/piper_ngi/local_process_tracking.py]: You gotta change this to be `extra_args=None` in the function definition, and then do a check in the code `if not extra_args: extra_args = {}`

Comment (mariogiov) [inline ngi_pipeline/engines/piper_ngi/local_process_tracking.py]: because mutable

File: ngi_pipeline/engines/piper_ngi/local_process_tracking.py [hunk 2]
```diff
@@ -359,7 +360,7 @@ def record_process_sample(project, sample, workflow_subtask, analysis_module_nam
                                                            project.base_path,
                                                            sample.name,
                                                            workflow_subtask)
-        recurse_status_for_sample(project_obj, "RUNNING", config=config)
+        recurse_status_for_sample(project_obj, "RUNNING", extra_args={'mean_autosomal_coverage':0},config=config)
```
Comment (mariogiov) [inline ngi_pipeline/engines/piper_ngi/local_process_tracking.py]: Spiffy!

File: ngi_pipeline/engines/piper_ngi/local_process_tracking.py (hunk 1 above)
Comment (guillermo-carrasco) [inline ngi_pipeline/engines/piper_ngi/local_process_tracking.py]: [reference](http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments)

**Pull request merged**